PhysicsTools/NanoAOD/python/particlelevel_cff.py

```diff
@@ -89,17 +89,6 @@
     src = rivetLeptonTable.src,
 )
 
-rivetMetTable = simpleSingletonCandidateFlatTableProducer.clone(
```

Comment:
Hi, I am not sure if any analysis actually uses it but the reason for including this fiducial MET definition was the following:

MiniAod contains packedGenParticles only up to eta 5, so when we calculate MissingMomentum in Rivet it full be different from the full GenMET. On the other hand, the fiducial MET is closer to what we actually measure, so it is a reasonable option to use this fiducial MET instead.

It could probably be renamed though, maybe `FiducialMET_pt/phi`.

Reply:
Hi @mseidel42, Just on the side note: Can't we use / apply this fiducial eta cut on the fly while using MET collection. In this way we can keep the centrally produced MET collection pretty simple. 

Secondly, to make the things consistent we should also save it for PuppiMET, right?

Reply:
Hi Wajid, any eta cut must be made on the input particles before calculating MET.

* Any reconstructed MET (PF, Puppi, Deep) is restricted by the detector coverage
* GenMET is the idealized case without any acceptance restriction 
* Fiducial MET is calculated at the generator level but restricted to the CMS coverage -> represents the best achievable MET estimate. This is also the only generator MET that can be recalculated from the genParticles in MiniAod.

Let me put a code suggestion here in a minute